ladybug_comfort/cli/utci.py

```diff
@@ -0,0 +1,364 @@
+# coding=utf-8
+"""Utility functions for calculating Universal Thermal Climate Index (UTCI).
+
+UTCI is a thermal comfort model strictly for the outdoors. It is an international
+standard for outdoor temperature sensation (aka. "feels-like" temperature) and
+is one of the most common of such "feels-like" temperatures used by
+meteorologists. UTCI that attempts to satisfy the following requirements:
+
+1) Thermo-physiological significance in the whole range of heat
+    exchange conditions of existing thermal environments
+2) Valid in all climates, seasons, and scales
+3) Useful for key applications in human biometeorology.
+
+While UTCI is designed to be valid in all climates and seasons, it assumes
+that human subjects are walking (with a metabolic rate around 2.4 met) and
+that they naturally adapt their clothing with the outdoor temperature.
+For outdoor situations that do not fit these criteria, the Physiological
+Equivalent Temperature (PET) model is recommended.
+"""
+import numpy as np
+
+
+def universal_thermal_climate_index(ta, tr, vel, rh):
+    """Calculate Universal Thermal Climate Index (UTCI) using a polynomial approximation.
+
+    This function here is a Python version of the original UTCI_approx
+    application written in Fortran. Version a 0.002, October 2009
+    The original Fortran code can be found at www.utci.org.
+
+    Note:
+        [1] Peter Bröde, Dusan Fiala, Krzysztof Blazejczyk, Yoram Epstein,
+        Ingvar Holmér, Gerd Jendritzky, Bernhard Kampmann, Mark Richards,
+        Hannu Rintamäki, Avraham Shitzer, George Havenith. 2009.
+        Calculating UTCI Equivalent Temperature. In: JW Castellani & TL
+        Endrusick, eds. Proceedings of the 13th International Conference
+        on Environmental Ergonomics, USARIEM, Natick, MA.
+
+    Args:
+        ta: Air temperature [C]
+        tr: Mean radiant temperature [C]
+        vel: Wind speed 10 m above ground level [m/s].
+            Note that this meteorological speed at 10 m is simply 1.5 times the
+            speed felt at ground in the original Fiala model used to build UTCI.
+        rh: Relative humidity [%]
```

Comment:
Do these have to be a specific data type that is different than the base function? So they can't just be floats or ints? If so, we should document this.

Reply:
You are right. These must be arrays, which is different than the base function.